specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-04-02-preview/managedClusters.json

```diff
@@ -4868,13 +4868,15 @@
       "type": "string",
       "enum": [
         "Ubuntu",
-        "CBLMariner"
+        "CBLMariner",
+        "Windows2019",
+        "Windows2022"
       ],
       "x-ms-enum": {
         "name": "OSSKU",
         "modelAsString": true
       },
-      "description": "Specifies an OS SKU. This value must not be specified if OSType is Windows."
```

Comment:
LGTM

So rp could handle `ossku = windows2019, ostype = linux` or other invalid combinations properly?

Reply:
Yes. AKS-RP validates the OSSKU value for Linux and Windows.

Reply:
Is there a default if one is not specified? For example `osType = Windows`, `osSKU` not set? If so we should document what that is (for both Windows/Linux)

Reply:
Good guidance. Updated.